celesta-core/src/main/java/ru/curs/celesta/dbutils/BasicCursor.java

```diff
@@ -1139,14 +1046,12 @@ boolean isEquivalent(BasicCursor c) throws CelestaException {
 			orderBy();
 		if (c.orderByNames == null)
 			c.orderBy();
-		if (orderByNames.length != c.orderByNames.length)
+
+		if (!Arrays.equals(orderByNames, c.orderByNames))
 			return false;
-		for (int i = 0; i < orderByNames.length; i++) {
-			if (!(orderByNames[i].equals(c.orderByNames[i])))
-				return false;
-			if (descOrders[i] != c.descOrders[i])
-				return false;
-		}
+		if (!Arrays.equals(descOrders, c.descOrders))
```

Comment:
Codacy is right here -- seems that we can do a single return here instead of if... block

Reply:
fixed